locales/tr/tr.json

```diff
@@ -1,618 +1,618 @@
     "A fresh verification link has been sent to your email address.": "Yeni bir doğrulama bağlantısı e-posta adresinize gönderildi.",
     "A new verification link has been sent to the email address you provided during registration.": "Üyelik esnasında kullandığınız e-posta adresinize onay linki gönderildi.",
-    "Accept Invitation": "Accept Invitation",
-    "Action": "Action",
-    "Action Happened At": "Happened At",
-    "Action Initiated By": "Initiated By",
-    "Action Name": "Name",
-    "Action Status": "Status",
-    "Action Target": "Target",
-    "Actions": "Actions",
+    "Accept Invitation": "Daveti Kabul Et",
+    "Action": "Aksiyon",
+    "Action Happened At": "Ne",
+    "Action Initiated By": "Tarafından Başlatılan",
+    "Action Name": "İsim",
+    "Action Status": "Durum",
+    "Action Target": "Hedef",
+    "Actions": "Hareketler",
     "Add": "Ekle",
     "Add a new team member to your team, allowing them to collaborate with you.": "Ekibinize yeni bir ekip üyesi ekleyerek, sizinle işbirliği yapmalarına izin verin.",
     "Add additional security to your account using two factor authentication.": "İki faktörlü kimlik doğrulama kullanarak hesabınıza ek güvenlik ekleyin.",
-    "Add row": "Add row",
+    "Add row": "Satır Ekle",
     "Add Team Member": "Ekip Üyesi Ekle",
     "Added.": "Eklendi.",
     "Administrator": "Yönetici",
     "Administrator users can perform any action.": "Yöneticiler herhangi bir eylemi gerçekleştirebilir.",
-    "Afghanistan": "Afghanistan",
-    "Aland Islands": "Åland Islands",
-    "Albania": "Albania",
-    "Algeria": "Algeria",
+    "Afghanistan": "Afganistan",
+    "Aland Islands": "Åland Adaları",
+    "Albania": "Arnavutluk",
+    "Algeria": "Cezayir",
     "All of the people that are part of this team.": "Bu ekibin parçası olan tüm kişiler.",
-    "All resources loaded.": "All resources loaded.",
+    "All resources loaded.": "Tüm kaynaklar yüklendi.",
     "All rights reserved.": "Tüm hakları saklıdır.",
     "Already registered?": "Zaten Üye Misiniz?",
-    "American Samoa": "American Samoa",
-    "An error occured while uploading the file.": "An error occured while uploading the file.",
+    "American Samoa": "Amerikan Samoası",
+    "An error occured while uploading the file.": "Dosya yüklenirken bir hata oluştu.",
     "Andorra": "Andorra",
     "Angola": "Angola",
     "Anguilla": "Anguilla",
-    "Another user has updated this resource since this page was loaded. Please refresh the page and try again.": "Another user has updated this resource since this page was loaded. Please refresh the page and try again.",
-    "Antarctica": "Antarctica",
-    "Antigua And Barbuda": "Antigua and Barbuda",
+    "Another user has updated this resource since this page was loaded. Please refresh the page and try again.": "Bu sayfa yüklendiğinden beri başka bir kullanıcı bu kaynağı güncelleştirdi. Lütfen sayfayı yenileyin ve tekrar deneyin.",
```

Comment:
`"Another user has updated this resource since this page was loaded. Please refresh the page and try again.": "Bu sayfa yüklendikten sonra başka bir kullanıcı sayfa kaynağını değiştirdi. Lütfen sayfayı yenileyin ve tekrar deneyin.",`